Zilon.Core/Zilon.Emulation.Common/InitializationBase.cs

```diff
@@ -95,9 +95,12 @@ private static void RegisterGameLoop(IServiceCollection serviceRegistry)
         /// </summary>
         private void RegisterAuxServices(IServiceCollection container)
         {
-            var linearDice = CreateRandomSeedAndLinearDice();
-            var gaussDice = CreateRandomSeedAndGaussDice();
-            var expDice = CreateRandomSeedAndExpDice();
+            // В комментариях - подавление ошибки от сервиса lgtm.com
+            // Это ложное срабатывание, потому что эти переменные используются в замыкании.
+            var linearDice = CreateRandomSeedAndLinearDice(); //lgtm [cs/useless-assignment-to-local]
+            var gaussDice = CreateRandomSeedAndGaussDice(); //lgtm [cs/useless-assignment-to-local]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'gaussDice'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5034101)